test/io/db_config.sql

```diff
@@ -63,6 +64,27 @@ ALTER ROLE other_authenticator SET pgrst.openapi_mode = 'disabled';
 ALTER ROLE other_authenticator SET pgrst.openapi_security_active = 'false';
 ALTER ROLE other_authenticator SET pgrst.server_trace_header = 'traceparent';
 
+create schema postgrest;
+grant usage on schema postgrest to db_config_authenticator;
+grant usage on schema postgrest to other_authenticator;
+
+-- pre-config hook
+create or replace function postgrest.pre_config()
+returns void as $$
+begin
+  if current_user = 'db_config_authenticator' then
+    null;
+  elsif current_user = 'other_authenticator' then
```

Comment:
```suggestion
  if current_user = 'other_authenticator' then
```

Or is the `db_config_authenticator` included to make it more explicit?

Reply:
Yeah, just for explicitness. But maybe it looks confusing, so I'd be fine with removing it from the conditional.